packages/recomponents/src/components/r-popper/r-popper.scss

```diff
@@ -1,3 +1,8 @@
+.r-popper-content-wrapper > *:only-child {
+    visibility: visible !important;
+    opacity: 1 !important;
+}
+
```

Comment:
for the `.dropdown`, `.popover` in Recomm, 
```html
visibility: hidden;
opacity: 0;
```
I need to change them to 
```html
visibility: visible;
opacity: 1;
```
but they are also used in other places, so I used !important here